src/components/common/ColonyActions/usePaginatedActions.ts

```diff
@@ -45,19 +50,23 @@ export const usePaginatedActions = (): UsePaginatedActionsReturn => {
     setSortDirection(newSortDirection);
   };
 
-  const loadMoreActions = () => {
+  if (hasMoreActions && filteredActions.length < actionsListSize) {
     fetchMore({
       variables: {
         nextToken,
       },
     });
+  }
+
+  const loadMoreActions = () => {
+    setCount((ct) => ct + 1);
```

Comment:
Instead of fetching directly from this call, we increase the amount of actions we want to display to the user (`actionListSize`). 

This will re-run the check on line 53. If we already have enough data in filtered actions, we don't need to fetch any more. Else, we will fetch more.